src/main/thrift/parquet.thrift

```diff
@@ -373,6 +462,56 @@ struct JsonType {
 struct BsonType {
 }
 
+/**
+ * Physical type and encoding for the geometry type.
+ */
+enum GeometryEncoding {
+  /**
+   * Allowed for physical type: BYTE_ARRAY.
+   *
+   * Well-known binary (WKB) representations of geometries. It supports 2D or
+   * 3D geometries of the standard geometry types (Point, LineString, Polygon,
+   * MultiPoint, MultiLineString, MultiPolygon, and GeometryCollection). This
+   * is the preferred option for maximum portability.
+   *
+   * This encoding enables GeometryStatistics to be set in the column chunk
+   * and page index.
+   */
+  WKB = 0;
```

Comment:
Should we specify between little-endian and big-endian.  The WKB specification terminology describes this as NDR (little-endian) or XDR (big-endian).  Im trying to come up with Iceberg spec and realize we didnt spec this here.  Im thinking NDR , unless we want to make it configurable

Reply:
cc @jiayuasu @paleolimbot sorry if already covered in existing discussion that I missed.

Reply:
@szehon-ho This was discussed above. WKB has the endianness specified in the first byte so both are allowed and readers will be able to identify and read it. So we don't need to specify that ourselves in Parquet and Iceberg spec